molecule/release-static/molecule.yml

```diff
@@ -22,28 +22,26 @@ lint: |
   capabilities:
     - NET_ADMIN
     - NET_RAW
+  pre_build_image: true
 platforms:
-  - <<: *platform_base
-    name: centos-7
-    image: centos:7
-    command: /sbin/init
   - <<: *platform_base
     name: centos-8
-    image: centos:8
+    image: pulp/molecule_centos8
```

Comment:
I went to look for a description of these images, but instead I did not find them on quay.io or dockerhub a tall.

Reply:
they are there:
https://hub.docker.com/repository/docker/pulp/molecule_fedora31
https://hub.docker.com/repository/docker/pulp/molecule_centos8
https://hub.docker.com/repository/docker/pulp/molecule_debian10

Reply:
They are all private:
https://hub.docker.com/search?q=pulp%2F&type=image
https://hub.docker.com/search?q=pulp%2Fmolecule_fedora31&type=image